.clang-format

```diff
@@ -0,0 +1,5 @@
+---
+Language: Cpp
+BasedOnStyle: Chromium
+IndentWidth: 8
```

Comment:
Please use:
```
TabWidth:        4
UseTab: Never
```

Reply:
This was copied from `prjxray`, so that needs updated as well.

Reply:
Done.